sum.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+#include <string.h> 
+
+int main(){
+	int nextvalue, z, x, q, runtotal, y;
+	int userinput[x], newarray[x];
```

Comment:
I think this is the main reason your code isn't working right now - again, you should include comments when your code isn't working!